HLTrigger/Configuration/python/HLT_75e33_cff.py

```diff
@@ -300,8 +300,7 @@
     fragment.HLT_Diphoton30_23_IsoCaloId_L1Seeded,
 
     fragment.HLT_DoubleMediumChargedIsoPFTauHPS40_eta2p1,
-    ### Removed temporarily until solution of https://github.com/cms-sw/cmssw/issues/42862
-    #fragment.HLT_DoubleMediumDeepTauPFTauHPS35_eta2p1,
+    fragment.HLT_DoubleMediumDeepTauPFTauHPS35_eta2p1,
```

Comment:
this should not be submitted here, but as a part of a backport of https://github.com/cms-sw/cmssw/pull/43855, otherwise it will lead to relval failures.

Reply:
removed